System_Template/Single_crystal_data/Readme.md

```diff
@@ -0,0 +1,25 @@
+Template file for recording single crystal SCXRD Experimental data. The file (csv) contains several columns:
+
+# Form (REFCODE)
+
+it is expected that this column will contain the common name of the polymorphic form, and if applicable, the CSD Refcode
```

Comment:
I was about to write a comment about whether REFCODE is the best name, given users might have data not in the CSD - Entry Identifier might be best.  Reading this though I am unsure what I would write - do I write both the polymorph name _and_ the refcode? If so how do I separate them and how does the reading software know which is which? I will suggest having separate columns, and or adding some examples so it is clear what to do in the cases where I have one, the other or both.